src/test/java/games/strategy/triplea/ui/UserActionPanelTest.java

```diff
@@ -23,53 +22,39 @@
 import games.strategy.engine.data.ResourceList;
 import games.strategy.triplea.Constants;
 import games.strategy.triplea.attachments.UserActionAttachment;
-import games.strategy.util.IntegerMap;
 
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
 public final class UserActionPanelTest {
   @Mock
   private GameData data;
 
   private Resource pus;
 
-  private Resource techTokens;
-
-  private Resource vps;
-
```

Comment:
These resources are not required in this fixture.  This allowed simplification of a lot of the related code.